pkg/apis/app/v1alpha1/kogitoinfra_types.go

```diff
@@ -116,11 +94,9 @@ const (
 // infrastructure dependency (such as Infinispan) to a third party operator.
 // It holds the deployment status of each infrastructure dependency and custom
 // resources needed to run Kogito Runtime and Kogito Data Index services.
+// KafkaTopics to be created for supporting services if KogitoInfra relates to a Kafka kind
```

Comment:
I believe we need to rethink this description, what about:

> KogitoInfra is the resource to bind a Custom Resource (CR) not managed by Kogito Operator to a given deployed Kogito service. It holds the reference of a CR managed by another operator such as Strimzi. For example: one can create a Kafka CR via Strimzi and link this resource using KogitoInfra to a given Kogito service (custom or supporting, such as Data Index). Please refer to the Kogito Operator documentation (https://docs.jboss.org/kogito/release/latest/html_single/) for more information.

Reply:
yes..its more clear